From 555ac92637a4343511a38cd4ff70d1aab6284d60 Mon Sep 17 00:00:00 2001
From: Holger Freyther <ich@tamarin.(none)>
Date: Tue, 13 May 2008 18:53:40 +0100
Subject: [PATCH] [pcf50633] Add exported symbol to the header file
     Reading the battery volt using the ADC was an exported symbol
     but not in the header file, add it.

Signed-Off-By: Holger Freyther <zecke@openmoko.org>
---
 include/linux/pcf50633.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/linux/pcf50633.h b/include/linux/pcf50633.h
index f427985..05b55f3 100644
--- a/include/linux/pcf50633.h
+++ b/include/linux/pcf50633.h
@@ -64,6 +64,9 @@ pcf50633_charge_enable(struct pcf50633_data *pcf, int on);
 extern void
 pcf50633_backlight_resume(struct pcf50633_data *pcf);
 
+extern u_int16_t
+pcf50633_battvolt(struct pcf50633_data *pcf);
+
 #define PCF50633_FEAT_EXTON	0x00000001	/* not yet supported */
 #define PCF50633_FEAT_MBC	0x00000002
 #define PCF50633_FEAT_BBC	0x00000004	/* not yet supported */
-- 
1.5.6.5

